react-form-controls/my-app/src/RegistrationFormControlled.js

```diff
@@ -0,0 +1,39 @@
+import { useState } from "react"
+
+export default function RegistrationFormControlled() {
+  let [usernameEntry, setUsernameEntry] = useState('');
```

Comment:
_Always_ declare state as `const`. This will keep you from modifying its value and thinking that might actually have an effect.